datastore/postgres/registerscanners.go

```diff
@@ -82,5 +83,60 @@ SELECT
 		registerScannerDuration.WithLabelValues("insert").Observe(time.Since(start).Seconds())
 	}
 
+	return s.populateScanners(ctx)
```

Comment:
is it assumed that `RegisterScanners` is called only once and in the very beginning of the process?